presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -142,18 +153,25 @@ Binary Functions
 
     Computes the 64-bit SpookyHashV2 hash of ``binary``.
 
+.. function:: xxhash64(binary) -> varbinary
+
+    Computes the xxhash64 hash of ``binary``.
```

Comment:
I believe it should be `xxHash64`

Reply:
There doesn't seem to be consensus, even from the author, but I agree that the capitalized version looks nicer.